cloudify/utils.py

```
@@ -200,18 +201,39 @@ def get_rest_cert_content():
     return os.environ[constants.REST_CERT_CONTENT_KEY]
 
 
+def _get_current_context():
+    for context in [workflow_ctx, ctx]:
+        try:
+            return context._get_current_object()
+        except RuntimeError:
+            continue
```

maybe raise an exception if we are not in the context for some unknown reason?